projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1456,3 +1460,13 @@ function publicize_calypso_url() {
 	_deprecated_function( __METHOD__, '0.2.0', 'Publicize::publicize_connections_url' );
 	return Redirect::get_url( 'calypso-marketing-connections', array( 'site' => ( new Status() )->get_site_suffix() ) );
 }
+
+/**
+ * Adds support for the post-thumbnails feature, regardless of underlying theme support.
+ *
+ * This ensures the featured image UI appears in the editor, allowing the user to
+ * explicitly set an image for their social media post.
+ */
+function add_theme_post_thumbnails_support() {
+	add_theme_support( 'post-thumbnails', get_post_types_by_support( 'publicize' ) );
+}
```

Comment:
Could this be in the class, instead of living outside of it?

Reply:
Discussed this one above https://github.com/Automattic/jetpack/pull/23871#discussion_r846856938
Making it a non-member function makes it easier for someone to use `remove_action` to remove this behaviour if that's not the way they want their theme to work. That was the idea anyway, maybe there's another way to allow people to override this hacking in of `post-thumbnails` support?